### PR TITLE
refactor(tx-details): revamp + dark theme

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Title says

##### Ticket

YOMO-1662

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

**Comment by banklesss** [comment]
base is the tx history PR

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/components/Modal/ModalScreen.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ const useStyles = () => {
     },
     sheet: {
       flex: 1,
-      backgroundColor: color.gray_c50,
+      backgroundColor: isDark ? color.gray_c50 : color.white_static,
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: exception

Comment (stackchain) [inline apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: @banklesss this shouldn't happen, can you check with Design why there is no subtoken for this case?

Comment (stackchain) [reply apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: maybe is just legacy....

Comment (banklesss) [reply apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: i asked them and they want a specifically white background in light mode and that color in dark

Comment (banklesss) [reply apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: 🤔

Comment (banklesss) [reply apps/wallet-mobile/src/components/Modal/ModalScreen.tsx:111]: i will ask them again

**Pull request merged**